tests/c20_client/test_client.py

```diff
@@ -1,17 +1,103 @@
-
 import requests_mock
-from c20_client.c20_client import do_job
+import pytest
+from c20_client.client import do_job
+from c20_client.connection_error import NoConnectionError
+
+
+def test_client_calls_decuments_endpoint_for_documents_job():
```

Comment:
I believe this is supposed to say calls_documents not calls_decuments. I am guessing that is a spelling mistake.